.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/metal"]
```

Comment:
I'm not sure to be honest, this was a convenient way to do it for now.  It would probably make more sense to have both gtest and metal work the same way whichever way we choose.